packages/manager/apps/pci-kubernetes/src/api/data/plugins.test.ts

```diff
@@ -0,0 +1,23 @@
+import { v6 } from '@ovh-ux/manager-core-api';
+import { vi } from 'vitest';
+import { updateAdmissionPlugin } from './plugins';
+
+describe('updateAdmissionPlugin', () => {
```

Comment:
I'm not sure about the interest of this unit test because it's an implementation detail.

If we consider testing the behavior under the `updateAdmissionPlugin` function, it will be better to test it through the `useUpdateAdmissionPlugin` hook. 

The SUT will be the behavior, like we do in BDD.